site/src/pages/TemplateVersionPage/TemplateVersionPage.test.tsx

```diff
@@ -38,10 +36,9 @@ const setup = async () => {
 describe("TemplateVersionPage", () => {
   beforeEach(setup)
 
-  it("shows the tf and md files only", () => {
+  it("shows files", () => {
     expect(screen.queryByText(TERRAFORM_FILENAME)).toBeInTheDocument()
     expect(screen.queryByText(README_FILENAME)).toBeInTheDocument()
```

Comment:
Why did we end up changing this test?

Reply:
Because now, the filtering happens inside of the "parsing" function. I made it to not have to parse 100 files to only get 2 of them after.